src/hardware/opl.cpp

```diff
@@ -872,13 +903,18 @@ OPL::OPL(Section *configuration, const OplMode oplmode)
 	if (dual_opl)
 		channel_features.emplace(ChannelFeature::Stereo);
 
-	mixer_chan = MIXER_AddChannel(OPL_CallBack, 0, "FM", channel_features);
+	const auto mixer_callback = std::bind(&OPL::AudioCallback,
+	                                      this,
+	                                      std::placeholders::_1);
+
+	// Register the Audio channel
+	channel = MIXER_AddChannel(mixer_callback, 0, "FM", channel_features);
```

Comment:
Btw, I've been thinking about renaming "FM" to "OPL", what do you think? We could do the usual deprecation warning, of course, and keep the "FM" alias (maybe forever, or at least for a very long time). It's just quite inconsistent with the rest of the channel names (FM is a synthesis technology), and the relevant config options are all prefixed with `opl`.

Reply:
Absolutely agree! I will certainly appreciate seeing "OPL" vs. "CMS". :rocket: